src/main/kotlin/no/nav/familie/ef/iverksett/behandlingstatistikk/BehandlingstatistikkRepository.kt

```diff
@@ -0,0 +1,39 @@
+package no.nav.familie.ef.iverksett.behandlingstatistikk
+
+import no.nav.familie.ef.iverksett.util.queryForJson
+import no.nav.familie.eksterne.kontrakter.saksstatistikk.ef.BehandlingDVH
+import no.nav.familie.kontrakter.ef.iverksett.Hendelse
+import no.nav.familie.kontrakter.felles.objectMapper
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate
+import org.springframework.stereotype.Repository
+import java.util.UUID
+
+@Repository
+class BehandlingstatistikkRepository(private val namedParameterJdbcTemplate: NamedParameterJdbcTemplate) {
+
+    fun lagre(behandlingId: UUID, behandlingDVH: BehandlingDVH, hendelse: Hendelse) {
+        return lagreBehandlingstatistikk(behandlingId, behandlingDVH, hendelse)
+    }
+
+    fun hent(behandlingId: UUID, hendelse: Hendelse): BehandlingDVH? {
+        val sql = "SELECT behandling_dvh FROM behandling_statistikk WHERE behandling_id = :behandlingId AND hendelse= :hendelse "
```

Comment:
Hvis vi skal hente data så kan det sikkert være fint å sette behandling_id som PRIMARY KEY i migreringsscriptet

Reply:
Ja,  må kanskje nesten legge til det, vi trenger flere innslag, sånn at for en behandlingsid, så har vi flere hendelser. Det gjøres oppslag på behandlingsid og hendelsestype. I værste fall, så vil det bli to innslag av samme hendelse akkurat nå

Reply:
(men må ha en én til mange relasjon evt da)

Reply:
Hvis en sak går litt "frem og tilbake" (underkjent i totrinnskontroll) - kan vi da risikere å få flere like hendelser på en behandling?

Reply:
Fint @throndi 
@charliemidtlyng Bra spørsmål :(

Reply:
Har du noen tanker om Charlie sitt spørsmål, @throndi?